xml/System.Net.Http/HttpClient.xml

```diff
@@ -566,12 +566,18 @@ The `DefaultRequestVersion` property can be changed as long as the <xref:System.
   
 ## Remarks  
  This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the whole response (including content) is read.  
-  
+
+> [!NOTE]
+> In case of timeout different types of exceptions are thrown on different .NET platform versions.
+> - HttpRequestException is thrown on all applicable versions of the .NET Framework
+> - TaskCancelledException without any inner exception is thrown on all applicable .NET Core versions
+> - TaskCancelledException nesting TimeoutException is thrown on .NET 5
  ]]></format>
         </remarks>
         <exception cref="T:System.ArgumentNullException">The <paramref name="requestUri" /> is <see langword="null" />.</exception>
         <exception cref="T:System.InvalidOperationException">The request message was already sent by the <see cref="T:System.Net.Http.HttpClient" /> instance.</exception>
         <exception cref="T:System.Net.Http.HttpRequestException">The request failed due to an underlying issue such as network connectivity, DNS failure, server certificate validation or timeout.</exception>
```

Comment:
If this exception is only thrown on .NET Framewok, then please prefix with ".NET Framework only: ".

Reply:
No, HttpRequestException still can be thrown on .NET Core due to other reasons besides timeout.